documentation/debugging.md

```diff
@@ -0,0 +1,7 @@
+# Debugging DigiTrust
+
+A live system may be changed to debug mode in one of two ways: via conf object or via the url hash `dt-debug=true` value.
```

Comment:
can you provide an example via conf object

Reply:
This markdown is incorrect - will need to revise. Had a design change to focus on log dumps as outlined here:
[](https://github.com/digi-trust/dt-cdn/wiki/Debugging-DigiTrust)

Reply:
Improved documents and aligned with actual code.